Assets/Scripts/GameController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class GameController : MonoBehaviour {
+
+  public GUIText GameOverText;
```

Comment:
I am learning this component now and will change the GUIText to UI Component later